readme.md

```diff
@@ -231,5 +273,16 @@ Convert the following ActiveRecord sequence to Mongoose:
 
 ```js
 // Your answer...
-```
+var andy = '';
+Instructor.find({name: "Andy"}).then(instructor) {
+  andy = instructor;
+});
+
+// not sure about this part 
+andy.insert({
```

Comment:
Close. Also, this action needs to take place inside the "then".  Otherwise, "andy" would not be assigned when this was called.  Please review the answer key.